shared/featureconfig/flags.go

```diff
@@ -156,6 +156,10 @@ var (
 		Name:  "init-sync-verbose",
 		Usage: "Enable logging every processed block during initial syncing. ",
 	}
+	enableFinalizedDepositsCache = &cli.BoolFlag{
```

Comment:
Add this to `beaconChainFlags` or else it wont come to affect. Most likely the reason why e2e fails:
https://github.com/prysmaticlabs/prysm/pull/6391/files#diff-0c0d18cd82b85b87dcc82b90036e884aR589

Reply:
Thanks! I missed this one when I was analysing an already merged feature flag PR.